metric_learn/rca.py

```diff
@@ -105,25 +105,30 @@ def fit(self, X, chunks):
                     ' It has been deprecated in version 0.5.0 and will be'
                     ' removed in 0.6.0. Use "n_components" instead',
                     DeprecationWarning)
+
+    if self.pca_comps != 'deprecated':
+      warnings.warn('"pca_comps" parameter is not used.'
+                    ' It has been deprecated in version 0.5.0 and will be'
+                    'removed in 0.6.0', DeprecationWarning)
+
     X, chunks = self._prepare_inputs(X, chunks, ensure_min_samples=2)
 
-    # PCA projection to remove noise and redundant information.
-    if self.pca_comps is not None:
-      pca = decomposition.PCA(n_components=self.pca_comps)
-      X_t = pca.fit_transform(X)
-      M_pca = pca.components_
-    else:
-      X_t = X - X.mean(axis=0)
-      M_pca = None
+    warnings.warn("RCA will no longer be trained on a preprocessed version "
```

Comment:
Do we need a ChangedBehaviorWarning? The default behavior was `pca_comps=None`
Maybe the information in this warning should rather go to the deprecation warning?

Reply:
see above